app/controllers/start_group_controller.rb

```diff
@@ -2,7 +2,9 @@ class StartGroupController < ApplicationController
   def new
     @errors = []
     @group = Group.new
-    if current_user.present? && ENV['NEW_GROUPS_ON_ANGULAR']
```

Comment:
Ah, ok, I'm with you now. So there's actually 3 paths

- logged in, angular enabled = redirect to dashboard
- logged in, angular not enabled = render enable_angular
- logged out, render new

Is that right?

I think dropping the ENV is still alright, and would like to move to always using `current_user_or_visitor.is_logged_in?` rather than `current_user.present?` for these controllers